### PR TITLE
Feature/1.1.1

### PR DESCRIPTION
**Pull request by fyang1024**
This is W-I-P. So far finished api-model and data-holder

### CODE REVIEW

File: client/src/main/java/au/org/consumerdatastandards/client/model/BankingProductV2.java [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+
+    private BankingProductAdditionalInformation additionalInformation;
+
+    private List<BankingProductV2CardArt> cardArt;
+
+    /**
+     * A data holder specific unique identifier for this product. This identifier must be unique to a product but does not otherwise need to adhere to ID permanence guidelines.
+     * @return productId
+     */
+    public String getProductId() {
```
Comment (vadkor) [inline client/src/main/java/au/org/consumerdatastandards/client/model/BankingProductV2.java]: No @Override annotations in V2 here and others, but they are there in V1...

File: client/src/main/java/au/org/consumerdatastandards/client/model/BankingScheduledPaymentRecurrenceLastWeekday.java [hunk 2]
```diff
@@ -47,27 +78,27 @@ public void setInterval(String interval) {
...
 
-    public void setLastWeekDay(Integer lastWeekDay) {
+    public void setLastWeekDay(LastWeekDay lastWeekDay) {
         this.lastWeekDay = lastWeekDay;
     }
 
     /**
-     * Indicates the number of payments remaining in the schedule. If both finalPaymentDate and paymentsRemaining are present then payments will stop according to the most constraining value. If neither field is present the payments will continue indefinitely
-     * @return paymentsRemaining
+     * Enumerated field giving the treatment where a scheduled payment date is not a business day. If absent assumed to be ON.&lt;br/&gt;**AFTER** - If a scheduled payment date is a non-business day the payment will be made on the first business day after the scheduled payment date.&lt;br/&gt;**BEFORE** - If a scheduled payment date is a non-business day the payment will be made on the first business day before the scheduled payment date.&lt;br/&gt;**ON** - If a scheduled payment date is a non-business day the payment will be made on that day regardless.&lt;br/&gt;**ONLY** - Payments only occur on business days. If a scheduled payment date is a non-business day the payment will be ignored
```
Comment (vadkor) [inline client/src/main/java/au/org/consumerdatastandards/client/model/BankingScheduledPaymentRecurrenceLastWeekday.java]: The <br/> shouldn't be escaped.

File: client/src/main/java/au/org/consumerdatastandards/client/api/BankingPayeesAPI.java [hunk 3]
```diff
@@ -38,7 +38,7 @@
      * http.response.details
      * <table summary="Response Details" border="1">
      *   <tr><td> Status Code </td><td> Description </td><td> Response Headers </td></tr>
-     *   <tr><td> ResponseCode.OK </td><td> Success </td><td>  -  </td></tr>
+     *   <tr><td> 200 </td><td> Success </td><td>  * x-v - The [version](#response-headers) of the API end point that the data holder has responded with. <br>  * x-fapi-interaction-id - An [RFC4122](https://tools.ietf.org/html/rfc4122) UUID used as a correlation id. If provided, the data holder must play back this value in the x-fapi-interaction-id response header. If not provided a [RFC4122] UUID value is required to be provided in the response header to track the interaction. <br>  </td></tr>
```
Comment (vadkor) [inline client/src/main/java/au/org/consumerdatastandards/client/api/BankingPayeesAPI.java]: Something is seriously wrong with this line here and in similar classes.

File: client/src/main/java/au/org/consumerdatastandards/client/model/BankingScheduledPaymentRecurrenceLastWeekday.java (hunk 2 above)
Comment (fyang1024) [reply client/src/main/java/au/org/consumerdatastandards/client/model/BankingScheduledPaymentRecurrenceLastWeekday.java]: fixed

File: client/src/main/java/au/org/consumerdatastandards/client/model/BankingProductV2.java (hunk 1 above)
Comment (fyang1024) [reply client/src/main/java/au/org/consumerdatastandards/client/model/BankingProductV2.java]: fixed

Comment (vadkor) [reply client/src/main/java/au/org/consumerdatastandards/client/model/BankingProductV2.java]: Thanks, you fixed it in this V2 class, but other V2 classes, such as BankingProductV2Details, have the same problem.

File: client/src/main/java/au/org/consumerdatastandards/client/api/BankingPayeesAPI.java (hunk 3 above)
Comment (fyang1024) [reply client/src/main/java/au/org/consumerdatastandards/client/api/BankingPayeesAPI.java]: fixed

**Pull request merged**